api4/plugin_test.go

```diff
@@ -59,6 +60,25 @@ func TestPlugin(t *testing.T) {
 	CheckNoError(t, resp)
 	assert.Equal(t, "testplugin", manifest.Id)
 
+	t.Run("install plugin from URL with slow response time", func(t *testing.T) {
+
+		if testing.Short() {
+			t.Skip("skipping test to install plugin from a slow response server")
+		}
+
+		//Install from URL - slow server to simulate longer bundle download times
```

Comment:
nit: space after comment start
```suggestion
		// Install from URL - slow server to simulate longer bundle download times
```